Demo/Classes/MBProgressHUD+GlobalHUD.m

```diff
@@ -0,0 +1,103 @@
+//
+//  MBProgressHUD+GlobalHUD.m
+//  Remind101
+//
+//  Created by Rex Fenley on 11/9/13.
+//
+
+// This code is distributed under the terms and conditions of the MIT license.
+
+// Copyright (c) 2013 Remind101, Rex Fenley
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MBProgressHUD+GlobalHUD.h"
+
+#import <objc/runtime.h>
+
+@implementation MBProgressHUD (GlobalHUD)
+
+static void * const kHUDCountStorageKey = (void *)&kHUDCountStorageKey;
+static MBProgressHUD * globalHUD;
+
++ (void)renewGlobalHUD
+{
+    NSArray *windows = [[UIApplication sharedApplication] windows];
+    UIWindow *window = [windows lastObject];
```

Comment:
I did something similar in our app to simplify process of migrating to `MBProgressHUD` from a library we were using before. And I used just first window here. It looks like it's always same as `AppDelegate#window`. And if there is an alert presented - it will have its own window which will be at the end of this array.

Usually these are the windows I see:

```
(lldb) po [[UIApplication sharedApplication] windows];
<__NSArrayM 0x796d7430>(
<UIWindow: 0x7a1643b0; frame = (0 0; 768 1024); tintColor = UIDeviceRGBColorSpace 0.117647 0.705882 0.784314 1; gestureRecognizers = <NSArray: 0x7a050a00>; layer = <UIWindowLayer: 0x7a13eb00>>,
<UITextEffectsWindow: 0x7a19f2e0; frame = (0 0; 768 1024); opaque = NO; autoresize = W+H; layer = <UIWindowLayer: 0x7a19f5b0>>
)
(lldb) po [[[UIApplication sharedApplication] delegate] window]
<UIWindow: 0x7a1643b0; frame = (0 0; 768 1024); autoresize = W+H; tintColor = UIDeviceRGBColorSpace 0.117647 0.705882 0.784314 1; gestureRecognizers = <NSArray: 0x7a050a00>; layer = <UIWindowLayer: 0x7a13eb00>>

```